### PR TITLE
Update C# Tutorials for RC2

### LINKED ISSUE #419: Update the C# tutorials to target RC2
For instance http://dotnet.github.io/docs/tutorials/getting-started-with-csharp/console-webapiclient.html is not compiling anymore. We need these to be updated for RC2. 

/cc @BillWagner @cartermp @mairaw

### PR DESCRIPTION
**Pull request by BillWagner**
At the moment, all work except for the microservice. That's waiting for some ASP.NET updates.

Fixes #419

### CODE REVIEW

**Comment by BillWagner** [comment]
All these now work with the latest CLI and RC2 changes.

/cc @blackdwarf

File: docs/tutorials/getting-started-with-csharp/console-teleprompter.md [hunk 1]
```diff
@@ -193,7 +193,14 @@ to add that using statement at the top of file:
 using System.Threading.Tasks;
 ```
 
-Run the sample, and check the output.Now, each single word is printed,
```
Comment (mairaw) [inline docs/tutorials/getting-started-with-csharp/console-teleprompter.md]: Line 17: avoid using here as a link text. Suggestion: You can find the installation instructions on the 
 [Getting started](http://dotnet.github.io/getting-started/) page.

Comment (mairaw) [inline docs/tutorials/getting-started-with-csharp/console-teleprompter.md]: Suggestion for line 19:
The descriptions below use -> This tutorial uses

Comment (mairaw) [inline docs/tutorials/getting-started-with-csharp/console-teleprompter.md]: Line 44: wouldn't be more accurate to say "creates your application executable"?

Comment (mairaw) [inline docs/tutorials/getting-started-with-csharp/console-teleprompter.md]: Line 48: Open that file you your favorite text editor. -> Open that file with your favorite text editor.

Comment (mairaw) [inline docs/tutorials/getting-started-with-csharp/console-teleprompter.md]: Line 68: Suggestion: Remove that sentence about the direct link and change the previous paragraph like this
Copy the "[sampleQuotes.txt](https://github.com/dotnet/core-docs/blob/master/docs/tutorials/getting-started-with-csharp/console-teleprompter/sampleQuotes.txt)"
 file from the GitHub repository for this [sample](https://github.com/dotnet/core-docs/tree/master/docs/tutorials/getting-started-with-csharp/console-teleprompter) into your project directory.

Comment (mairaw) [inline docs/tutorials/getting-started-with-csharp/console-teleprompter.md]: Line 106: Enumerator Methods -> Enumerator methods

Comment (mairaw) [inline docs/tutorials/getting-started-with-csharp/console-teleprompter.md]: Line 126: Var defines -> `var` defines

Comment (mairaw) [inline docs/tutorials/getting-started-with-csharp/console-teleprompter.md]: Line 130: which is a `StreamReader` -> which is a `StreamReader` object.

Comment (mairaw) [inline docs/tutorials/getting-started-with-csharp/console-teleprompter.md]: nit: Line 312: remove comma after `GetInput()`

File: docs/tutorials/getting-started-with-csharp/microservices.md [hunk 2]
```diff
@@ -81,24 +81,23 @@ The template creates six files for you:
 * A wwwroot/web.config file. This contains basic configuration information.
 
 Now you can run the template generated application. That's done using a series
-of tools from the command line. The current version of ASP.NET uses two command
-line tools for building and running your applications: dnu and dnx. Dnu is the .NET Execution
-Environment Utility. Dnx is the .NET Execution Engine. 
+of tools from the command line. The `dotnet` command runs the tools necessary
+for .NET development. Each verb execute a different command
```
Comment (mairaw) [inline docs/tutorials/getting-started-with-csharp/microservices.md]: Each verb execute -> Each verb executes

File: docs/tutorials/getting-started-with-csharp/microservices.md [hunk 3]
```diff
@@ -81,24 +81,23 @@ The template creates six files for you:
...
+`dotnet build`
 
-And once you build the application, you run it from the command line using dnx:
+And once you build the application, you run it from the command line:
 
-`dnx web`
+`dotnet run`
 
 The default configuration listens to http://localhost:5000. You can open a
 browser and navigate to that page and see a "Hello World!" message.
```
Comment (mairaw) [inline docs/tutorials/getting-started-with-csharp/microservices.md]: Line 125: It's Main() method -> Its Main() method

Comment (mairaw) [inline docs/tutorials/getting-started-with-csharp/microservices.md]: nit: Line 166: missing space before Run

**Comment by mairaw** [comment]
Overall looks good. Left some comments. Didn't follow the steps personally just read them but can do that later on if needed.

**Comment by BillWagner** [comment]
This needs to wait a bit to merge, until there is a Newtonsoft.JSON that works with RC 2. (hopefully soon)

**Comment by BillWagner** [comment]
Merging, but see issue #462 to add new content (based on priority)
/cc @blackdwarf

**Pull request merged**